### PR TITLE
Jasonl/feat/update UI

### PR DESCRIPTION
**Pull request by sleeper-jasonl**
### Description
Adding toast and switch UI features. Also adjusting dependencies to fix yarn errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,32 +34,33 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@callstack/repack": "blitzstudios/repack.git#callstack-repack-v2.6.3-gitpkg",
+    "@callstack/repack": "2.6.3",
```
Comment (sleeper-luke) [inline package.json]: Just thinking aloud here - maybe it's time that we actually publish our forks of these projects on npm - that way we don't have to worry about version incompatibilities by devs not using the right branches. This is a bit ambiguous - both our forked gitpkg and the official 2.6.3 release would match.

**Review by sleeper-luke — APPROVED** [review]

**Review by sleeper-luke — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sleeper-jasonl) [reply package.json]: We probably should, but maybe when things settle a bit?

File: package.json [hunk 2]
```diff
@@ -34,32 +34,33 @@
...
+    "react-native-svg": "13.7.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
     "@types/lodash": "^4.14.182",
-    "@types/react-native": "0.66.5",
+    "@types/react-native": "0.67.7",
```
Comment (sleeper-luke) [inline package.json:63]: I believe this is intended to correspond to the current version of react native that we have. `0.66.5`. Do we need to update this?

Comment (sleeper-luke) [reply package.json:63]: Oooh, I see. It was updated recently in the main app. Interesting.

**Review by sleeper-luke — APPROVED** [review]

**Pull request merged**